app/services/appropriate_bodies/close_induction.rb

```diff
@@ -15,10 +15,14 @@ class TeacherHasNoOngoingInductionPeriod < StandardError; end
     attribute :author
 
     validates :finished_on, presence: { message: "Enter a finish date" }
+    validates :written_fail_confirmation_on, presence: { message: "Enter a date written confirmation of failed Induction" }, if: :failed_outcome?
     validates :number_of_terms, presence: { message: "Enter a number of terms" }
 
     def self.induction_params
-      { model_name.param_key => %i[finished_on number_of_terms] }
+      permitted = %i[finished_on number_of_terms]
+      permitted << :written_fail_confirmation_on if self == AppropriateBodies::RecordFail || self == Admin::RecordFail
```

Comment:
Doing it this way makes the parent responsible for its children. All these changes also impact `RecordPass` and `RecordRelease` which are unrelated. Better to instead move failure specific logic into the appropriate service, overwritting methods where necessary. The admin version can behave differently if needed and that'd be more intuitive.